### DISCUSSION TITLE
Does gluonts Trainer supports multi-GPU training?

### DISCUSSION THREAD

**Discussion by la-cruche** [awslabs/gluonts#2348] [Q&A] [answered]
From this old issue it seems no - https://github.com/awslabs/gluonts/discussions/1164 ; can you confirm there is still no multi-device nor multi-node data parallel support in GluonTS?

**Comment by lostella** [comment, marked as answer]
This is not supported in MXNet-based models (the training loop we use does not support that currently). For PyTorch models, we rely on Lightning for training, so there it may be possible (but I haven't done it, so this needs to be validated).
